.github/workflows/ci.yml

```
@@ -72,9 +72,9 @@ jobs:
       - name: Run Tests
         run: ./vendor/bin/phpunit --coverage-clover build/coverage/xml
 
-      - name: Upload coverage results to Coveralls
-        if: "${{ matrix.coverage != 'none' }}"
+      - name: Upload coverage results to Codacy
         env:
           CODACY_PROJECT_TOKEN: ${{ secrets.CODACY_PROJECT_TOKEN }}
+        if: "${{ matrix.coverage != 'none' && env.CODACY_PROJECT_TOKEN != '' }}"
```

> This fixes build failures on pull requests from forks...

You could instead check whether the PR is not from a fork:

```suggestion
        if: github.event.pull_request.head.repo.fork == false
```

---

I'd prefer it this way as this allows forks to still upload coverage to their own coverage server by setting the auth key.

---

Ah I see. That would include the person also using Codacy.

---

Indeed.